surveys/2024-annual-survey/translations/fr.md

```diff
@@ -605,22 +605,22 @@ Type: select all that apply
 
 Type: select all that apply
 
-- Qu'il n'y ait pas assez d'utilisation dans l'industrie
-- Qu'il y ait trop d'intérêt de la part des grandes entreprises
-- Qu'il n'y ait pas assez de contributions open source à l'écosystème
-- Qu'il n'ajoute pas une fonctionnalité spécifique que je veux
-- Qu'il n'évolue pas assez vite
-- Qu'il soit instable
-- Qu'il soit remplacé par une alternative
+- Qu’il n'y ait pas assez d'utilisation dans l'industrie
+- Qu’il y ait trop d'intérêt de la part des grandes entreprises
+- Qu’il n'y ait pas assez de contributions open source à l'écosystème
+- Qu’il n'ajoute pas une fonctionnalité spécifique que je veux
+- Qu’il n'évolue pas assez vite
+- Qu’il soit instable
+- Qu’il soit remplacé par une alternative
 - Que le langage ne devienne trop complexe
-- Que les outils et la documentation ne soient pas suffisamment accessibles (par exemple, en raison de la langue ou de l'incompatibilité avec les liseuses)
+- Que les outils et la documentation ne soient pas suffisamment accessibles (par exemple, en raison de la langue ou de l’incompatibilité avec les lecteurs d’écran)
```

Comment:
```suggestion
- Que les outils et la documentation ne soient pas suffisamment accessibles (par exemple, en raison de la langue ou de l’incompatibilité avec les liseuses)
```

Reply:
I changed this because "liseuse" refers more to things like the Kindle and Ko

Reply:
Oh, indeed. I didn't look at the original for that one, my bad.